internal/routes/liveupdates/seventv/basic.go

```diff
@@ -0,0 +1,67 @@
"height\":128,\"name\":\"4x.avif\",\"size\":4634,\"static_name\":\"4x\",\"width\":128},{\"format\":\"WEBP\",\"frame_count\":1,\"height\":128,\"name\":\"4x.webp\",\"size\":6122,\"static_name\":\"4x\",\"width\":128}],\"url\":\"//cdn.7tv.app/emote/621d13967cc2d4e1953838ed\"},\"id\":\"621d13967cc2d4e1953838ed\",\"lifecycle\":3,\"listed\":true,\"name\":\"Chatterinoge\",\"owner\":{\"avatar_url\":\"//cdn.7tv.app/pp/60ae3e98b2ecb0150535c6b7/4aa1786cec024098be20d7b0683bae72\",\"connections\":null,\"display_name\":\"gempir\",\"id\":\"60ae3e98b2ecb0150535c6b7\",\"roles\":[\"6076a86b09a4c63a38ebe801\"],\"style\":{\"color\":16755200,\"paint\":null},\"username\":\"gempir\"},\"tags\":[\"okayge\",\"chatterino\"]},\"flags\":0,\"id\":\"621d13967cc2d4e1953838ed\",\"name\":\"Chatterinoge\",\"timestamp\":1667059799105}}],\"pulled\":[{\"key\":\"emotes\",\"index\":224,\"type\":\"object\",\"old_value\":{\"actor_id\":\"60b39e943e203cc169dfc106\",\"flags\":0,\"id\":\"621d13967cc2d4e1953838ed\",\"name\":\"Chatterino\",\"timestamp\":-62135596800000},\"value\":null}],\"updated\":[{\"key\":\"emotes\",\"index\":224,\"type\":\"object\",\"old_value\":{\"actor_id\":\"60b39e943e203cc169dfc106\",\"flags\":0,\"id\":\"621d13967cc2d4e1953838ed\",\"name\":\"Chatterinoge\",\"timestamp\":1667059799105},\"value\":{\"actor_id\":\"60b39e943e203cc169dfc106\",\"data\":{\"animated\":false,\"flags\":0,\"host\":{\"files\":[{\"format\":\"AVIF\",\"frame_count\":1,\"height\":32,\"name\":\"1x.avif\",\"size\":1504,\"static_name\":\"1x\",\"width\":32},{\"format\":\"WEBP\",\"frame_count\":1,\"height\":32,\"name\":\"1x.webp\",\"size\":1052,\"static_name\":\"1x\",\"width\":32},{\"format\":\"AVIF\",\"frame_count\":1,\"height\":64,\"name\":\"2x.avif\",\"size\":2450,\"static_name\":\"2x\",\"width\":64},{\"format\":\"WEBP\",\"frame_count\":1,\"height\":64,\"name\":\"2x.webp\",\"size\":2462,\"static_name\":\"2x\",\"width\":64},{\"format\":\"AVIF\",\"frame_count\":1,\"height\":96,\"name\":\"3x.avif\",\"size\":3644,\"static_name\":\"3x\",\"width\":96},{\"format\":\"WEBP\",\"frame_count\":1,\"height\":96,\"name\":\"3x.webp\",\"size\":4334,\"static_name\":\"3x\",\"width\":96},{\"format\":\"AVIF\",\"frame_count\":1,\"height\":128,\"name\":\"4x.avif\",\"size\":4634,\"static_name\":\"4x\",\"width\":128},{\"format\":\"WEBP\",\"frame_count\":1,\"height\":128,\"name\":\"4x.webp\",\"size\":6122,\"static_name\":\"4x\",\"width\":128}],\"url\":\"//cdn.7tv.app/emote/621d13967cc2d4e1953838ed\"},\"id\":\"621d13967cc2d4e1953838ed\",\"lifecycle\":3,\"listed\":true,\"name\":\"Chatterinoge\",\"owner\":{\"avatar_url\":\"//cdn.7tv.app/pp/60ae3e98b2ecb0150535c6b7/4aa1786cec024098be20d7b0683bae72\",\"connections\":null,\"display_name\":\"gempir\",\"id\":\"60ae3e98b2ecb0150535c6b7\",\"roles\":[\"6076a86b09a4c63a38ebe801\"],\"style\":{\"color\":16755200,\"paint\":null},\"username\":\"gempir\"},\"tags\":[\"okayge\",\"chatterino\"]},\"flags\":0,\"id\":\"621d13967cc2d4e1953838ed\",\"name\":\"Chatterino\",\"timestamp\":1667059799105}}]}}}"
```

Comment:
Would be nice to use go raw string literals here if it's not too much of a hassle